_posts/2022-10-06-hard-mode-rust.adoc

```diff
@@ -133,7 +133,7 @@ image::https://user-images.githubusercontent.com/1711539/194287665-05583649-dcb0
 This works rather intuitive conceptually.
 First, imagine the above scene, with an infinite fuchsia colored plane and a red Utah teapot hovering above that.
 Then, imagine a camera standing at `0,10,-50` (in cartesian coordinates) and aiming at the origin.
-Now, draw an imaginary rectangular 80x60 screen at a focus distance of 50 from the camera along its line of side.
```

Comment:
This is intended to be 80x60 -- this is in scene's real dimensions, not in the screan pixels. In other words, one pixel is of the size 0.1 in the scene. If the focal distance is just 50, than 800x600 would be way to gigantic screen.

Reply:
oh, in that case, it might be nice to note that in the post